### PR TITLE
edit card , adding Reham.html

### PR DESCRIPTION
**Pull request by RehamFarhat**
solve issue #5

Hi Alaa

### CODE REVIEW

File: Reham.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            margin-top: 20px;
+        }
+    </style>
+</head>
+<body>
+    <h1>Imagine</h1>
+    <h2>John Lennon</h2>
+    <div class="lyric">
+        <p>Imagine lyric
+blalallalalallala</p>
```
Comment (AlaaJb99) [inline Reham.html:29]: change the text

**Review by AlaaJb99 — COMMENTED** [review]
in Reham.html

File: Reham.html (hunk 1 above)
Comment (RehamFarhat) [reply Reham.html]: Ok , changed.

**Review by RehamFarhat — COMMENTED** [review]
see the changes

**Pull request merged**